### DISCUSSION TITLE
HW 2 Part 3 Q1

### DISCUSSION THREAD

**Discussion by JeffHann** [cooperrc/computational-mechanics#98] [Q&A]
Follow up to my question from earlier in the livestream so I can attach pictures. I understand to plot them together (the yellow and red lines show the individual power consumption for the US and Europe separately), my question was whether we should plot the sum of the US and Europe's power consumption (the blue line) and then use that for the linear regressions for parts b and c?

Plot for Parts A and B: 
<img width="350" alt="image" src="https://user-images.githubusercontent.com/112503497/191794691-6762b53e-29e6-490c-a225-845299b92c5e.png">

 
Plot for Part C:
<img width="352" alt="image" src="https://user-images.githubusercontent.com/112503497/191794743-1c412b13-47df-4cef-a8c8-ea80b9c49e64.png">

**Comment by cooperrc** [comment]
Its interesting to see them added together. 

I was originally thinking to keep the data separate, but plotted on the same graph.
